scripts/attendance.py

```diff
@@ -0,0 +1,167 @@
+            return hours, minutes, available_working_days
+        else:
+            tt = (timedelta(hours=average_attendace_hours) +
+                  timedelta(minutes=int(average_attendace_minutes))) - \
+                 timedelta(hours=DAILY_WORKING_HOURS)
+            mins = self.time_to_minutes(tt)
+            td = timedelta(minutes=mins * present_days)
+            days, hours, minutes = td.days, td.seconds // 3600, td.seconds //\
+                60 % 60
+            extra_work = timedelta(hours=days * 24) + timedelta(hours=hours) +\
+                timedelta(minutes=minutes)
+            retData = (timedelta(days=available_working_days) - extra_work) / \
+                timedelta(hours=9)
+            result_list = str(retData).split(".")
+            hours, minutes = result_list[0], result_list[1][:2]
+            return hours, minutes, available_working_days
+
+
+def is_validate_time_format(average_attendace):
+    """
+    This will check is user entered time is valid or not
+    :param average_attendace:
+    :return:
+    """
+    timeformat = "%H:%M:%S"
+    try:
+        valid_time = datetime.strptime(average_attendace, timeformat)
+    except ValueError:
+        print("Please enter average present time in hh:mm:ss format")
+        attendance()
+
+
+def is_valid_integer(number, error_message):
+    """
+    This will check is user entered value is valid integer or not
+    :param number:
+    :param error_message:
+    :return:
+    """
+    try:
+        valid_number = input(number)
+    except ValueError:
+        print(error_message)
+        attendance()
+
+
+def attendance():
+    attendance_management = AttendanceManagement()
+    average_attendace = input("Entaer your average attendance in 'hh:mm:ss'"
+                              " format. E.g. 8:00:00  ")
+    is_validate_time_format(average_attendace)
+
+    present_days = int(input("Enter your present days. E.g. 19  "))
+    is_valid_integer(present_days, "Please neter valid INTEGER value for"
+                                   " present days")
+
+    leaves = int(input("Enter your leaves taken in current month. E.g. 2  "))
+    is_valid_integer(present_days, "Please neter valid INTEGER value for"
+                                   " leaves ")
+
+    available_working_days = \
+        attendance_management.get_total_working_days_remaining(
+            attendance_management.current_month,
+            attendance_management.current_year, present_days)
+    available_working_days = available_working_days - leaves
+    hours, minutes, available_working_days = \
+        attendance_management.get_avarage_working_target(
+            average_attendace, available_working_days, present_days)
+
+    print("you have to seat {hours} hours and {minutes} minutes"
+          " for next {days}"
+          " days ".format(hours=hours, minutes=minutes,
+                          days=available_working_days))
+
+
+if __name__ == '__main__':
+    attendance()
```

Comment:
How about adding Unit tests?
Unit testing improves the quality of the code
Help to find the usecases that we missed while implementing.
and most important the main benefits of unit testing is that it makes the coding process more Agile